multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -24,6 +24,10 @@ externalDNSName: {{.ExternalDNSName}}
 # it for you.
 #hostedZone: ""
 
+# The ID of hosted zone to add the externalDNSName to.
+# Only necessary if there are multiple hosted zones with the same name
+#hostedZoneId: ""
```

Comment:
Actually, I misread. So you want to keep both configurables so that the common case (by name) is easier to specify.

I would lean toward trying to get people to use the ID exclusively. This could change over time (a new hosted zone with same name is added later) - and then how does this tool react on re-use (e.g. upgrades)? 

Is it difficult to extract the hostedZoneID?